api-docs/api-reference/methods/token/get-list-endpoints-for-token-v2.0-tokens-tokenid.rst

```diff
@@ -0,0 +1,185 @@
+.. _get-list-endpoints-for-token-v2.0-tokenid:
+
+List endpoints for token
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    GET /v2.0/tokens/{tokenId}/endpoints
+
+Returns a list of endpoints points associated with a specific token.
+
+This call returns a list of endpoints associated with a specific token. Like
+the service catalog returned by a successful authentication, you can use this
+list of endpoints as the basis of a list of the services available to this
+user.
+
+
+The following table shows the possible response codes for this operation:
+
++-------------------------+-------------------------+-------------------------+
```

Comment:
We currently stay away from these types of tables since they are hard to modify. Here is an example of what we use: https://raw.githubusercontent.com/rackerlabs/docs-cloud-identity/master/api-docs/api-reference/methods/identity-provider/get-identity-provider-metadata-v2.0.rst